app/src/main/res/layout-sw600dp-land/onboarding_fragment.xml

```diff
@@ -70,23 +70,24 @@
           android:id="@+id/onboarding_fragment_next_image_view"
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
-          android:layout_marginEnd="20dp"
+          android:layout_marginEnd="32dp"
           android:gravity="center"
           android:minWidth="48dp"
           android:minHeight="48dp"
           android:onClick="@{(v) -> presenter.clickOnNext()}"
-          android:visibility="@{viewModel.slideNumber != 3 ? View.VISIBLE: View.GONE, default=visible}"
           android:paddingStart="12dp"
```

Comment:
Remove this padding start and padding end.

Instead you have to use `marginStart = 12dp` and `marginEnd = 12dp`

Also, just an explanation of how you can easily cross-verify out these things

The bottom bar layout height is `72dp` in mocks.
The height of image button arrow is `48dp`
The margin top for arrow is `12dp`as per mocks
The margin bottom for arrow is `12dp` as per mocks

72dp = 48dp + 12dp + 12dp

Reply:
Done